maestro-cli/src/main/java/maestro/cli/App.kt

```diff
@@ -19,29 +19,16 @@
 
 package maestro.cli
 
-import maestro.cli.command.BugReportCommand
-import maestro.cli.command.CloudCommand
-import maestro.cli.command.DownloadSamplesCommand
-import maestro.cli.command.LoginCommand
-import maestro.cli.command.LogoutCommand
-import maestro.cli.command.PrintHierarchyCommand
-import maestro.cli.command.QueryCommand
-import maestro.cli.command.RecordCommand
-import maestro.cli.command.StudioCommand
-import maestro.cli.command.TestCommand
-import maestro.cli.command.UploadCommand
+import maestro.cli.command.*
```

Comment:
I personally prefer having all the imports explicitly in the file since IntelliJ IDEA does a good job of hiding them from me and we avoid using the wrong dependency.

Do we have any style rules to this? If not, feel free to ignore my comment.

Reply:
Agree - Updated my IDE settings to not use star imports and updated the PR